### ISSUE TITLE
integrate a web pdf viewer

### ISSUE THREAD

**Issue by mgogoulos** [mediacms-io/mediacms#84]
for viewing PDF online. Eg https://github.com/webodf/ViewerJS

**Comment by swiftugandan** [comment]
+1

**Comment by mgogoulos** [comment]
https://github.com/mozilla/pdf.js

**Comment by mgogoulos** [comment]
Moreover a pdf icon could be displayed as poster for pdfs (on listings)

**Comment by amifunny** [comment]
@mgogoulos I would like to take on this feature.

**Comment by swiftugandan** [comment]
@mgogoulos Just wondering, doesn't this need the frontend/react code?

**Comment by amifunny** [comment]
@swiftugandan @mgogoulos Yes, it would require React/FrontEnd code base to develop this feature. As mentioned in this issue - https://github.com/mediacms-io/mediacms/issues/7 there was a plan to release it as seperate project in `mediacms-io`. Is there a plan to release it soon? I am gsoc student looking to make this as my proposal project. It would be helpful to upload frontend codebase as well.

**Comment by mgogoulos** [comment]
Hi, we're doing some refactoring on the code before it can be released, the plan is to make the release during April.

**Comment by mgogoulos** [comment]
Frontend code is now available, so we can work on this!
